exporting/mongodb/metadata.yaml

```diff
@@ -0,0 +1,151 @@
+# yamllint disable rule:line-length
+---
+id: 'export-mongodb'
+meta:
+  name: 'MongoDB'
+  link: 'https://www.mongodb.com/'
+  categories:
+    - export
+  icon_filename: 'json.png'
+keywords:
+  - exporter
+  - MongoDB
+overview:
+  exporter_description: |
+    Use the MongoDB connector for the exporting engine to archive your agent's metrics to a MongoDB database
+    for long-term storage, further analysis, or correlation with data from other sources.
+  exporter_limitations: ''
+setup:
+  prerequisites:
+    list:
+      - title: ''
+        description: |
+          - To use MongoDB as an external storage for long-term archiving, you should first [install](http://mongoc.org/libmongoc/current/installing.html) libmongoc 1.7.0 or higher.
+          - Next, re-install Netdata from the source, which detects that the required library is now available.
+  configuration:
+    file:
+      name: 'exporting.conf'
+    options:
+      description: |
+        The following options can be defined for this exporter.
+      folding:
+        title: 'Config options'
+        enabled: true
+      list:
+        - name: 'enabled'
+          default_value: 'no'
+          description: 'Enables or disables an exporting connector instance (yes|no).'
+          required: true
+        - name: 'destination'
+          default_value: 'pubsub.googleapis.com'
+          description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+          required: true
+          detailed_description: |
+            The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+            - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+            - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+            - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+
+            Example IPv4:
```

Comment:
We are not using MongoDB [default port](https://www.mongodb.com/docs/manual/reference/default-mongodb-port/) here, please, modify it.

Reply:
These are generic for config options.